Makefile

```diff
@@ -1,4 +1,4 @@
-JSX_TESTS := $(shell find ./test \! -name "._*.js" -name "*_test.js")
+JSX_TESTS := $(shell find ./test \! -name "._*.jsx" -name "*_test.jsx")
```

Comment:
it doesn't look like this variable is used in the Makefile? The `test` target appears to invoke mocha on the whole test directory. Can probably just remove this line